WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostRenderer.java

```diff
@@ -307,6 +310,9 @@ private void replaceIframeTag(final String tag, final String src) {
      * returns the full content, including CSS, that will be shown in the WebView for this post
      */
     private String formatPostContentForWebView(final String content, boolean renderAsTiledGallery) {
```

Comment:
Cool, what do you have in mind Nick? Something like searching for `tiled-gallery` in the content html? It has to be something simple and accurate.